documentation/api/exceptions.md

```diff
@@ -3,7 +3,7 @@
 
 # Exceptions History - Get
 
-Captures a history of first chance exceptions that were thrown in the [default process](defaultprocess.md).
+Captures a history of first chance exceptions that were thrown in the specified process.
```

Comment:
I think this just fell through the cracks from the recent behavior change - it doesn't specifically apply to this change, but I figured I'd update it here.